$MirOS: ports/net/wget/patches/patch-doc_wget_texi,v 1.3 2005/11/10 22:20:43 tg Exp $
--- doc/wget.texi.orig	Sat Oct 14 23:26:50 2006
+++ doc/wget.texi	Sat Oct 14 23:26:50 2006
@@ -11,6 +11,11 @@
 @setchapternewpage on
 @c %**end of header
 
+@dircategory Networking tools
+@direntry
+* Wget: (wget.info).            A utility for network download.
+@end direntry
+
 @iftex
 @c Remove this if you don't use A4 paper.
 @afourpaper
@@ -187,7 +192,7 @@ Built-in features offer mechanisms to tu
 The progress of individual downloads is traced using a progress gauge.
 Interactive downloads are tracked using a ``thermometer''-style gauge,
 whereas non-interactive ones are traced with dots, each dot
-representing a fixed amount of data received (1KB by default).  Either
+representing a fixed amount of data received (1 KiB by default).  Either
 gauge can be customized to your preferences.
 
 @item
@@ -761,7 +766,7 @@ seconds.
 Limit the download speed to @var{amount} bytes per second.  Amount may
 be expressed in bytes, kilobytes with the @samp{k} suffix, or megabytes
 with the @samp{m} suffix.  For example, @samp{--limit-rate=20k} will
-limit the retrieval rate to 20KB/s.  This is useful when, for whatever
+limit the retrieval rate to 20 KiB/s.  This is useful when, for whatever
 reason, you don't want Wget to consume the entire available bandwidth.
 
 This option allows the use of decimal numbers, usually in conjunction
